templates/dockerdiscordcontrol.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>DockerDiscordControl</Name>
+  <Repository>dockerdiscordcontrol/dockerdiscordcontrol</Repository>
```

Comment:
Please specify an explicit default tag, even if it is the implicit `latest` tag.